tests/test_sentry.py

```diff
@@ -59,18 +59,7 @@
                             "module": "falcon.app",
                             "post_context": ANY,
                             "pre_context": ANY,
-                            "vars": {
-                                "__builtins__": "<module 'builtins' (built-in)>",
-                                "__doc__": "'Falcon App class.'",
-                                "__file__": ANY,
-                                "__loader__": ANY,
-                                "__name__": "'falcon.app'",
-                                "__package__": "'falcon'",
-                                "__spec__": ANY,
-                                "iscoroutinefunction": ANY,
-                                "re": "<module 're' from '/usr/local/lib/python3.9/re.py'>",
-                                "wraps": ANY,
-                            },
```

Comment:
I don't know how the sentry package extracts the local variables, but my guess is they copy `f_locals` from the frame object. That has always been one of the weirder corners of Python, or at least since local variables aren't dictionaries anymore, so since the 2.0 release? Updates to `f_locals` are not reflected in local variables, and updates to local variables are not always reflected in `f_locals` as well.

I'm not surprised that this kind of change can happen when upgrading Python, and I don't think we care.